src/simulators/state.hpp

```diff
@@ -242,6 +242,12 @@ class State {
   void snapshot_creg_register(const Operations::Op &op, ExperimentResult &result,
                               std::string name = "register") const;
 
+  //add final state to result
+  virtual void add_state_to_data(ExperimentResult &result)
```

Comment:
Shouldn't we have overrides of this function in other files such as `state_chunk.hpp`, `statevector_state_chunk.hpp`, etc.

Reply:
sorry I forgot to remove this, removed